Base/Python/slicer/util.py

```diff
@@ -1188,6 +1188,39 @@ def __enter__(self):
   def __exit__(self, type, value, traceback):
     self.node.EndModify(self.wasModifying)
 
+
+
+#
+# Subject hierarchy
+#
+def getSubjectHierarchyItemChildren(parentItem=None, recursive=False):
+  """Convenience method to get children of a subject hierarchy item.
+  :param vtkIdType parentItem: Item for which to get children for. If omitted
+         or None then use scene item (i.e. get all items)
+  :param bool recursive: Whether the query is recursive. False by default
+  :return: List of child items
+  """
+  children = []
+  shNode = slicer.mrmlScene.GetSubjectHierarchyNode()
+  # Use scene as parent item if not given
+  if not parentItem:
+    parentItem = shNode.GetSceneItemID()
+  childrenIdList = vtk.vtkIdList()
+  shNode.GetItemChildren(parentItem, childrenIdList, recursive)
+  for childIndex in range(childrenIdList.GetNumberOfIds()):
+    children.append(childrenIdList.GetId(childIndex))
+  return children
+
+def getSubjectHierarchyItemAllChildren(parentItem=None):
```

Comment:
Do we really need this? Users could simply call `getSubjectHierarchyItemChildren(parentItem, True)` themselves.

Reply:
I added this because if we call it without any argument then we get all SH items. If you want I can remove it.

Reply:
I would vote for removing it, just to keep the API a bit smaller (it is already quite big, and growing).

Reply:
OK one minute.